core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/PathElement2d.java

```diff
@@ -36,935 +36,940 @@
  */
 public abstract class PathElement2d implements PathElement2afp {
```

Comment:
I really do not know what happened here. Probably a bug in Source tree. The class file has been replaced by itself.